core/helper_api.php

```diff
@@ -694,3 +694,20 @@ function helper_filter_by_prefix( array $p_set, $p_prefix ) {
 	return $t_matches;
 }
 
+/**
+ * Build the query part of an url, with an array of parameter name => value pairs
+ * This function does not prepends '?' to the resulting query string
+ * Parameter names and values must be valid for inclusion in a query string.
+ * @param array $p_params	Array of name => value pairs
+ * @return string	Resulting query string
+ */
+function helper_url_query_string( array $p_params ) {
```

Comment:
Thanks, i knew there was something already implemented, but couldn't find at that moment...